assets/js/base/components/chip/stories/chip.stories.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { Story, Meta } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import Chip, { ChipProps } from '../chip';
+const availableElements = [ 'li', 'div', 'span' ];
+const availableRadius = [ 'none', 'small', 'medium', 'large' ];
+
+export default {
+	title: 'WooCommerce Blocks/@base-components/Chip',
+	component: Chip,
+	argTypes: {
+		element: {
+			control: 'select',
+			options: availableElements,
+		},
+		className: {
+			control: 'text',
+		},
+		radius: {
+			control: 'select',
+			options: availableRadius,
+		},
+		children: {
+			control: 'text',
```

Comment:
Are chips really supposed to have children? I thought they were self-closing elements with the `text` attribute. In fact, when you edit this control as text, it makes things pretty weird. What do you think?

Reply:
I know what you mean, setting children here is not really possible, whatever the type of control. I think I added this just to try to parse a html string `<div>child</div>` but gave up on the idea. The `Chip` component does render its' children and it is a valid prop, but I will remove this control here and just leave the default in place